docs/source/ops/common-metrics.rst

```diff
@@ -8,6 +8,27 @@ The below sections contain the list of common metrics we expose for Daml service
 
 For the metric types referenced below, see the `relevant Prometheus documentation <https://prometheus.io/docs/tutorials/understanding_metric_types/>`_.
 
+Health Metrics
+**************
+
+We expose the below metrics for each components.
+
+daml_health_status
+==================
+
+- **Description**: The status of the component
+- **Values**:
+
+  - **0**: Not healthy
+  - **1**: Healthy
+  
+- **Labels**: 
+
+  - **component**: the name of the component being monitored
+
+- **Type**: Gauge
+
+
 gRPC Metrics
 ************
 We expose the below metrics for all gRPC endpoints.
```

Comment:
```suggestion
We expose the following metrics for all gRPC endpoints.
```

Reply:
I was following the existing pattern, but it looked strange to me.
I have put the clean up of the `below`s in another PR, which will likely be merge first -> #16099